config/config_test.go

```diff
@@ -1296,6 +1296,47 @@ func TestConfigJSONMarshalRoundtrip(t *testing.T) {
 				PackagePath: "path/to/home/depot",
 			},
 		},
+		{
+			name: "traffic tunnel endpoints",
+			c: config.Config{
+				Network: config.NetworkConfig{
+					NetworkConfigData: config.NetworkConfigData{
+						TrafficTunnelEndpoints: []config.TrafficTunnelEndpoint{
+							{
+								Port:              9090,
+								ConnectionTimeout: 20 * time.Second,
+							},
+							{
+								Port:              27017,
+								ConnectionTimeout: 40 * time.Millisecond,
```

Comment:
maybe add one with no/0 connection timeout?

Reply:
Does the `TrafficTunnelEndpoint` right below this one capture what you're looking for? Just a specified port so we use a zero-valued `time.Duration` (`0`) for `ConnectionTimeout`?

Reply:
oh yep, you had added it and I definitely just missed that.